ReleaseNotes-2.11

```diff
@@ -69,3 +69,4 @@ Breaking API changes:
 =====================
 
  * Status messages endpoints. Check the API documentation for the new format.
+ * Announcement outputs have changed. Check the API documentation for the new format.
```

Comment:
No need to mention the changed output. Mention that  `/announcement` is a deprecated API and will be removed in the next version.